### PR TITLE
Make combined flow the default

### PR DESCRIPTION
**Pull request by jknapp25**
Flip the default from separate flow to combined (prescribe + order) to aid in the migration toward combined.

### CODE REVIEW

File: packages/settings/src/lib/neutron.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ const defaultSettings: OrganizationSettings = {
   returnTo: window.location.origin,
   federated: false,
   enableMedHistory: false,
-  enableRxAndOrder: false
+  enableRxAndOrder: true
```
Comment (jknapp25) [inline packages/settings/src/lib/neutron.ts:30]: neutron default setting

File: packages/settings/src/lib/photon.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ const defaultSettings: OrganizationSettings = {
   returnTo: window.location.origin,
   federated: false,
   enableMedHistory: false,
-  enableRxAndOrder: false
+  enableRxAndOrder: true
```
Comment (jknapp25) [inline packages/settings/src/lib/photon.ts:28]: photon default setting

**Review by marisbest2 — APPROVED** [review]

File: packages/settings/src/lib/neutron.ts [hunk 3]
```diff
@@ -64,46 +63,51 @@ export const neutron: {
...
+    sendOrder: false,
+    enableRxAndOrder: false
   },
   // Modern Pediatrics
   org_XiV8H4uCFu6QFUov: {
     ...defaultSettings,
     logo: 'modern_pediatrics_logo.png',
-    accentColor: '#3f7a9c'
+    accentColor: '#3f7a9c',
+    enableRxAndOrder: false // doublecheck
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:79]: drop

**Review by captDaylight — APPROVED** [review]

File: packages/settings/src/lib/neutron.ts [hunk 4]
```diff
@@ -64,46 +63,51 @@ export const neutron: {
...
   },
   // Summer Health
   org_jr5fT3Mh4zCFsVQA: {
     ...defaultSettings,
     logo: 'summer_health_logo.svg',
-    accentColor: '#ffc21f'
+    accentColor: '#ffc21f',
+    enableRxAndOrder: false // doublecheck
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:86]: drop

File: packages/settings/src/lib/neutron.ts [hunk 5]
```diff
@@ -64,46 +63,51 @@ export const neutron: {
...
   },
   // Modern Ritual
   org_0WP4tWCftMEM7K2q: {
     ...defaultSettings,
     logo: 'modern_ritual_logo.webp',
     accentColor: '#202a36',
-    sendToPatient: false
+    sendToPatient: false,
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:94]: drop

File: packages/settings/src/lib/neutron.ts [hunk 6]
```diff
@@ -64,46 +63,51 @@ export const neutron: {
...
   },
   // River Health
   org_U3ofDUVRsNTYt7d8: {
     ...defaultSettings,
     logo: 'river_health_logo.svg',
     accentColor: '#2faef3',
     mailOrder: true,
-    mailOrderProviders: [CAREPOINT_ID]
+    mailOrderProviders: [CAREPOINT_ID],
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:110]: drop

**Review by captDaylight — APPROVED** [review]

File: packages/settings/src/lib/neutron.ts [hunk 7]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
+    accentColor: '#141d45',
+    enableRxAndOrder: false
   },
   // Assure Health
   org_RPrHGTLipYfPywxf: {
     ...defaultSettings,
     logo: 'assure_health_logo.svg',
-    accentColor: '#66cca9'
+    accentColor: '#66cca9',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:197]: drop

File: packages/settings/src/lib/neutron.ts [hunk 8]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
   },
   // Sunrise
   org_SgWtqCKFzYaDePCf: {
     ...defaultSettings,
     logo: 'sunrise_logo.svg',
-    accentColor: '#0057b8'
+    accentColor: '#0057b8',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:204]: drop

File: packages/settings/src/lib/neutron.ts [hunk 9]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
+    accentColor: '#1c3f28',
+    enableRxAndOrder: false
   },
   // Miga Health
   org_u3kKBHGj4MFfzOVz: {
     ...defaultSettings,
     logo: 'miga_health_logo.svg',
-    accentColor: '#12110c'
+    accentColor: '#12110c',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:218]: drop

File: packages/settings/src/lib/neutron.ts [hunk 10]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
+    accentColor: '#5224C7',
+    enableRxAndOrder: false
   },
   // Oshi Health
   org_yOgsgGMBVUZIBcwp: {
     ...defaultSettings,
     logo: 'oshi_logo.svg',
-    accentColor: '#3b5b80'
+    accentColor: '#3b5b80',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:232]: drop

File: packages/settings/src/lib/neutron.ts [hunk 11]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
+    accentColor: '#12110c',
+    enableRxAndOrder: false
   },
   // Sesame
   org_QFoulY6Ornx7dMdw: {
     ...defaultSettings,
     logo: 'sesame_logo.jpg',
-    accentColor: '#5224C7'
+    accentColor: '#5224C7',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:225]: drop

File: packages/settings/src/lib/neutron.ts [hunk 12]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
+    accentColor: '#3b5b80',
+    enableRxAndOrder: false
   },
   // TBD Health
   org_kI58h1e86jYO37LQ: {
     ...defaultSettings,
     logo: 'tbd_logo.svg',
-    accentColor: '#ee8155'
+    accentColor: '#ee8155',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:239]: drop

File: packages/settings/src/lib/neutron.ts [hunk 13]
```diff
@@ -114,119 +118,138 @@ export const neutron: {
...
   },
   // Openloop Health
   org_t5lbHt9eb9gMeeeb: {
     ...defaultSettings,
     logo: 'openloop_logo.png',
-    accentColor: '#E90C54'
+    accentColor: '#E90C54',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/neutron.ts:246]: drop

File: packages/settings/src/lib/photon.ts [hunk 14]
```diff
@@ -40,48 +40,51 @@ export const photon: {
...
+    accentColor: '#ffc21f'
   },
   // Modern Ritual
   org_IWmodVdrQYpqh5x2: {
     ...defaultSettings,
     logo: 'modern_ritual_logo.webp',
     accentColor: '#202a36',
-    sendToPatient: false
+    sendToPatient: false,
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:64]: drop

File: packages/settings/src/lib/photon.ts [hunk 15]
```diff
@@ -40,48 +40,51 @@ export const photon: {
...
   },
   // Reside Health
   org_8fwLMYQGxj6Bg70X: {
     ...defaultSettings,
     logo: 'reside_health_logo.webp',
-    accentColor: '#0c3276'
+    accentColor: '#0c3276',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:71]: drop

File: packages/settings/src/lib/photon.ts [hunk 16]
```diff
@@ -40,48 +40,51 @@ export const photon: {
...
   },
   // Radish Health
   org_tHroj0b67F08q1yw: {
     ...defaultSettings,
     logo: 'red_radish_logo.svg',
-    accentColor: '#ba4a71'
+    accentColor: '#ba4a71',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:78]: drop

File: packages/settings/src/lib/photon.ts [hunk 17]
```diff
@@ -40,48 +40,51 @@ export const photon: {
...
   },
   // River Health
   org_jScrLol7ZMSfExSR: {
     ...defaultSettings,
     logo: 'river_health_logo.svg',
     accentColor: '#2faef3',
     mailOrder: true,
-    mailOrderProviders: [CAREPOINT_ID]
+    mailOrderProviders: [CAREPOINT_ID],
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:87]: drop

File: packages/settings/src/lib/photon.ts [hunk 18]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
+    sendToPatient: false,
+    enableRxAndOrder: false
   },
   // ZZPeds
   org_pxcJ7Dlclcsg0OJw: {
     ...defaultSettings,
     logo: 'zzpeds_logo.png',
-    accentColor: '#5271ff'
+    accentColor: '#5271ff',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:106]: drop

File: packages/settings/src/lib/photon.ts [hunk 19]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
   },
   // MisterRx
   org_Frco5TQQFDHqS7xY: {
     ...defaultSettings,
     pickUp: false,
     mailOrder: true,
     mailOrderProviders: [CUREXA_ID],
-    sendToPatient: false
+    sendToPatient: false,
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:115]: drop

File: packages/settings/src/lib/photon.ts [hunk 20]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
   },
   // Mishe
   org_kLRbIfgsTXHmXDcj: {
     ...defaultSettings,
     logo: 'mishe_logo.svg',
-    accentColor: '#652D92'
+    accentColor: '#652D92',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:122]: drop

File: packages/settings/src/lib/photon.ts [hunk 21]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
   },
   // Bold Health
   org_TVIFBLp7zkkDGeQI: {
     ...defaultSettings,
     logo: 'bold_health_logo.svg',
-    accentColor: '#25532b'
+    accentColor: '#25532b',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:129]: drop

File: packages/settings/src/lib/photon.ts [hunk 22]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
+    sendToPatient: false,
+    enableRxAndOrder: false
   },
   // Ognomy
   org_2B1yxV6yS9ROqV1g: {
     ...defaultSettings,
     logo: 'ognomy_logo.webp',
-    accentColor: '#0a7ade'
+    accentColor: '#0a7ade',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:170]: drop

File: packages/settings/src/lib/photon.ts [hunk 23]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
   },
   // Carbon Health
   org_Dye8T9VAM1GHjjzS: {
     ...defaultSettings,
     logo: 'carbon_health_logo.svg',
-    accentColor: '#000000'
+    accentColor: '#000000',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:177]: drop

File: packages/settings/src/lib/photon.ts [hunk 24]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
-    enableRxAndOrder: true
+    enableMedHistory: true
   },
   // TBD Health
   org_XoBVNLkIWL6BP8vZ: {
     ...defaultSettings,
     logo: 'tbd_logo.svg',
-    accentColor: '#ee8155'
+    accentColor: '#ee8155',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:198]: drop

File: packages/settings/src/lib/photon.ts [hunk 25]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
   },
   // Openloop Health
   org_Oxc0CSPfdiyWW3VM: {
     ...defaultSettings,
     logo: 'openloop_logo.png',
-    accentColor: '#E90C54'
+    accentColor: '#E90C54',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:205]: drop

File: packages/settings/src/lib/photon.ts [hunk 26]
```diff
@@ -92,108 +95,120 @@ export const photon: {
...
   },
   // PM Pediatrics Care
   org_2Mqu8Kf7dknKNFXb: {
     ...defaultSettings,
     logo: 'pm_pediatrics_logo.svg',
-    accentColor: '#4D15B7'
+    accentColor: '#4D15B7',
+    enableRxAndOrder: false
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:212]: drop

**Pull request merged**